content/app.md

```diff
@@ -30,6 +30,36 @@ Galaxy records the output of your app's logs in addition to printing operational
 
 You can [filter logs from a specific container](/containers.html#connect-logs) on the app container page.
 
+<h3 id="icon-meanings">Icons</h3>
+
+Icons in your app's dashboard will display the current state of your app.
+
+<img src="/images/green_circle.png" style="margin: 1em 0;"/>
+
+The green full circle indicates that your app's container set is running, and passed its health checks. For an app that should be running, this is the desired state.
+
+<img src="/images/green_circle_outlined.png" style="margin: 1em 0;"/>
+
+The outlined green full circle means that your app's container set status is starting, updating, or stopping. It should leave this state fairly quickly.
+
+<img src="/images/empty_circle_outlined.png" style="margin: 1em 0;"/>
+
+The outlined green empty circle means that your app's container set has experienced an issue while starting.
+
+This can mean:
```

Comment:
This means both of these simultaneously, but I think the way you word it makes it sound like one or the other.

I think it is best to be really explicit that this only happens on your app's initial deploy, or starting it after it was stopped.